src/about.js

```diff
@@ -25,32 +25,26 @@ ${getValaVersion()}
 ${getBlueprintVersion()}
 `.trim();
 
-  const dialog = new Gtk.AboutDialog({
-    application,
-    authors: ["Sonny Piers https://sonny.re"],
-    comments: _("Learn and prototype with\nGNOME technologies"),
+  const dialog = new Adw.AboutWindow({
+    application_name: "Workbench",
+    developer_name: "Sonny Piers",
+    comments: _("Learn and prototype with GNOME technologies"),
```

Comment:
if we remove "comments" - I think the "details" row is replaced with the website row directly which I would prefer

Reply:
Correct.